routes/chat.js

```diff
@@ -1,33 +1,35 @@
 /**
  * Created by einavcarmon on 20/06/2017.
  */
+const { onlySuperuser } = require('./helpers')
```

Comment:
hmm maybe put all 'require's in the same place? not sure how that mechanism works so can't really say which one's faster (but pretty sure there's no functional difference since a module has its own scope)